### DISCUSSION TITLE
Creating Android apps

### DISCUSSION THREAD

**Discussion by rvwilliams88** [unoplatform/uno#16961] [General]
I'm just starting to learn about the Uno Platform and have completed the Simple Calculator tutorial using C# and MVUX. One of my reasons for learning how to use the platform is so that I can run apps on my Android phone. What I'm clear about is the process for creating an Android app. I have compiled the Simple Calculator for release but the net8-android folder is empty. What would I need to do to create an Android version of Simple Calculator and install it on my phone?

**Comment by jeromelaban** [comment]
There are multiple ways to push your app to your phone, but the best way to do it is to use https://platform.uno/docs/articles/uno-publishing-android.html.

Then you can use `adb install your.apk` to get your app installed on your phone.
